cmd/config.go

```diff
@@ -111,22 +120,42 @@ func defaultConfig(keyHome string, debug bool) []byte {
 		modules = append(modules, v)
 	}
 	cfg := Config{
-		Chain: &client.ChainClientConfig{
-			Key:            "default",
-			ChainID:        "cosmoshub-4",
-			RPCAddr:        "https://cosmoshub-4.technofractal.com:443",
-			GRPCAddr:       "https://gprc.cosmoshub-4.technofractal.com:443",
-			AccountPrefix:  "cosmos",
-			KeyringBackend: "test",
-			GasAdjustment:  1.2,
-			GasPrices:      "0.01uatom",
-			KeyDirectory:   keyHome,
-			Debug:          debug,
-			Timeout:        "20s",
-			OutputFormat:   "json",
-			BroadcastMode:  "block",
-			SignModeStr:    "direct",
-			Modules:        modules,
+		DefaultChain: "cosmoshub-4",
+		Chains: []*client.ChainClientConfig{
+			{
+				Key:            "default",
+				ChainID:        "cosmoshub-4",
+				RPCAddr:        "https://cosmoshub-4.technofractal.com:443",
+				GRPCAddr:       "https://gprc.cosmoshub-4.technofractal.com:443",
+				AccountPrefix:  "cosmos",
+				KeyringBackend: "test",
+				GasAdjustment:  1.2,
+				GasPrices:      "0.01uatom",
+				KeyDirectory:   keyHome,
+				Debug:          debug,
+				Timeout:        "20s",
+				OutputFormat:   "json",
+				BroadcastMode:  "block",
```

Comment:
should avoid using block, this was meant for testing that ended up bleeding into prod.

Reply:
oh interesting

Reply:
I guess I should add a mempool query that sees if we have a TX in the mempool before trying to submit another one